scripts/sync-and-publish

```diff
@@ -22,8 +22,8 @@ case $opt in
         ;;
 esac
 
-aptly mirror create ${release}-mirror https://mattermost-repository-deb.s3.us-east-1.amazonaws.com ${release}
-aptly mirror update ${release}-mirror
+aptly mirror -ignore-signatures create ${release}-mirror https://mattermost-repository-deb.s3.us-east-1.amazonaws.com ${release} 
+aptly mirror -ignore-signatures update ${release}-mirror
```

Comment:
Release.gpg is generated with expired public key. Cause of it aptly mirror is not working. After first successful run we will revert those changes cause Release.gpg is generated with new public key.